scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -254,43 +254,62 @@ it is useful e.g. when having multiple lanes with the same name, but with differ
 }
 
 export class LaneChangeScopeCmd implements Command {
-  name = 'change-scope <remote-scope-name>';
+  name = 'change-scope <lane-name-or-remote-scope> <remote-scope-name>';
```

Comment:
@benjgil this is confusing.
Now it looks like I have to pass a remote scope or scope name and a scope name.
Please change it back and write the second option in the comment below (if you want to rename a scope name that you are not checked to ... you can use a first argument with the lane name

Reply:
the examples should make it apparent what this does. 
this does exactly what you're suggesting, so i'm not sure what your issue is with how this works?

Reply:
I think this can be resolved by changing the brackets type. According to @davidfirst a [] brackets is optional, so i'll use that for the second argument